<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QCheckBox" name="cB_Output">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cB_KeepOnTop">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>KeepOnTop</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>230</width>
     <height>180</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="com">
    <attribute name="title">
     <string>COM</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>BaudRate</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbox_baudRate">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbox_comName">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_comOpen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_comClose">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="hid">
    <attribute name="title">
     <string>HID</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="other">
    <attribute name="title">
     <string>Other</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>66</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Key Display</string>
   </property>
   <widget class="QLabel" name="lb_shift">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Shift</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_keyValue">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>KeyValue:None</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_win">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>35</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Win</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_alt">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Alt</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ctrl">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>35</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Ctrl</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_keyName">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>35</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>KeyName:None</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cB_HoldOn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>HoldOn</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cB_MapNUM">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>MapNUM</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
